jmeter/jmeter_plugins.txt

```diff
@@ -0,0 +1,66 @@
+blazemeter-debugger=0.6
+jpgc-directory-listing=0.3
+bzm-random-csv=0.7
+bzm-parallel=0.11
+jpgc-rotating-listener=0.2
+jpgc-sense=3.5
+jpgc-wsc=0.7
+bzm-hls=3.1
+bzm-http2=1.6.1
+bzm-rte=3.2
+jmeter-core=0.0
```

Comment:
do we need all plugins with 0 versions? what does 0.0 version mean?

Reply:
The 0.0 version plugins are JMeter default plugins. Since they are installed and enabled on a fresh JMeter installation I removed them from the list. Thank you.